<launch>
	

  <node name="stability_publisher" pkg="robbie_stability" type="robbie_stability_ex" respawn="false" output="screen"/>

  <node name="height_publisher" pkg="robbie_auto" type="height_publisher" respawn="false" output="screen"/>

  <node name="contact_publisher" pkg="robbie_stability" type="robbie_contact_ex" respawn="false" output="screen"/>

</launch>


